cli/src/commands/log.rs

```diff
@@ -78,7 +78,8 @@ pub(crate) fn cmd_log(
 
     let revset_expression = {
         let mut expression = if args.revisions.is_empty() {
-            workspace_command.parse_revset(&command.settings().default_revset())?
+            workspace_command
+                .parse_revset(&RevisionArg::from(command.settings().default_revset()))?
```

Comment:
nit: undo accidental (?) formatting change

Reply:
sorry, which one?

this could be `.into()` instead of `RevisionArg::from()` btw.

Reply:
Oh, sorry! That was just because my browser window was a little too narrow so it looked like this added a blank line here.